### PR TITLE
idasen: init at 0.7.1

### PR DESCRIPTION
**Pull request by newAM**
###### Motivation for this change

Add [idasen](https://github.com/newAM/idasen)

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by newAM** [comment]
Whoops, sorry about that CI failure, I accidentally included a line I should not have when I was staging.

File: pkgs/development/python-modules/idasen/default.nix [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+, pythonOlder
+, fetchFromGitHub
+, bleak
+, pyyaml
+, voluptuous
+, pytest
+, pytest-asyncio
+, poetry-core
+}:
+buildPythonPackage rec {
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/idasen/default.nix:12]:
```suggestion
}:

buildPythonPackage rec {
```

File: pkgs/development/python-modules/idasen/default.nix [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  pname = "idasen";
+  version = "0.7.1";
+  doCheck = true;
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/idasen/default.nix:15]:
```suggestion
```

thats the default

File: pkgs/development/python-modules/idasen/default.nix [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+  ];
+
+  propagatedBuildInputs = [
+    bleak
+    pyyaml
+    voluptuous
+  ];
+
+  checkInputs = [
+    pytest
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/idasen/default.nix:38]:
```suggestion
    pytestCheckHook
```

File: pkgs/development/python-modules/idasen/default.nix [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+    pytest-asyncio
+  ];
+
+  checkPhase = "pytest";
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/idasen/default.nix:42]:
```suggestion
```

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]

**Comment by newAM** [comment]
Made the requested changes, thanks for the review!

**Pull request merged**